packages/@expo/server/tsconfig.mjs.json

```diff
@@ -0,0 +1,9 @@
+{
+  "extends": "./tsconfig.json",
+  "compilerOptions": {
+    "moduleResolution": "node",
+    "module": "es2022",
+    "target": "es2022",
```

Comment:
We should keep this at `esnext`. The problem here is that we either have to specify inlining the helpers for transpilation, or be aware of them (in our case, we wouldn't want them to be a dep). But in practice, `es2022` is a very wide support range and we should just avoid going beyond it when writing code. However, there's no good linter for that (was on my todo list at some point)

tl;dr though: let's keep this on `target: 'esnext'` like the base/cjs output